frontend/plausible_health_check.py

```diff
@@ -0,0 +1,22 @@
+import sys
+
+import requests
```

Comment:
`requests` isn't standard library and not everyone will have it installed in their global python installation (I don't, for example). `urlopen` is a better choice here, but less convenient, if we stick with a python script instead of a shell-based approach.

+1 to Dhruv's comment that try/except is just fine here, totally appropriate usage.